source/fab/steps/archive_objects.py

```diff
@@ -11,6 +11,7 @@
 import logging
 from string import Template
 from typing import Optional
+from pathlib import Path
```

Comment:
I would recommend to sort the imports alphabetically within the same group of imports (group as in: system, ..., local projects). 
I don't think it's part of Fab's coding style, so I'll let this pass ... but if the modules are sorted, it's easier to see if you are importing a module twice.

Reply:
I am happy to ignore this, given that sorting the imports alphabetically (i.e. pathlib before typing) is not part of our coding standards :)